client/src/config/contribution-guidelines.md

```diff
@@ -25,6 +29,10 @@ We will accept submissions of 10x Genomics single-cell or single-nuclei profilin
 Researchers must process their data using the Childhood Cancer Data Lab's production pipeline – available at <https://github.com/AlexsLemonade/scpca-nf/> – and submit the output, along with project (see [**Submitting Project Metadata**](#submitting-project-metadata)), sample (see [**Submitting Sample Metadata**](#submitting-sample-metadata)) and cell (see [**Submitting Cell Metadata**](#submitting-cell-metadata)) metadata.
 Submitters must also sign a Data Transfer Agreement before data transfer ([view Data Transfer Agreement template](https://docs.google.com/document/d/1eLHobiV4M0bC0KOQALgpfaLU0Y4nMZYs/edit?usp=share_link&ouid=105890053693989014850&rtpof=true&sd=true)).
 
+<form>
```

Comment:
Let's replace this with either a comment or a link and ask Jackie to update the main copy.

If using a comment:
- do a normal string replace with the form html that below this comment

If using a link:
- override the createElement and find the link you want to replace with a button